### PR TITLE
🐛 Follow type changes from `layoutShift` to `layout-shift`.

### PR DESCRIPTION
**Pull request by ericandrewlewis**
The PerformanceObserver API has changed in the spec and Chromium, this commit follows those changes.

See:
 - https://github.com/WICG/layout-instability/commit/cada7215d5be4a4b83cac363a2980327dcbe6a86#diff-04c6e90faac2675aa89e2176d2eec7d8
 - https://chromium-review.googlesource.com/c/chromium/src/+/1685975

### CODE REVIEW

**Comment by ericandrewlewis** [comment]
I realized there's a few other changes in the spec other than this - going to follow-up with a few more commits.

File: src/service/performance-impl.js [hunk 1]
```diff
@@ -301,8 +301,12 @@ export class Performance {
         recordedFirstInputDelay = true;
       } else if (entry.entryType === 'layoutJank') {
         this.aggregateJankScore_ += entry.fraction;
-      } else if (entry.entryType === 'layoutShift') {
-        this.aggregateShiftScore_ += entry.value;
+      } else if (entry.entryType === 'layout-shift') {
```
Comment (philipwalton) [inline src/service/performance-impl.js]: Do you also need `layoutShift` here? Without it,  I think you'll miss entries from Chrome 76.

File: src/service/performance-impl.js [hunk 2]
```diff
@@ -301,8 +301,12 @@ export class Performance {
         recordedFirstInputDelay = true;
       } else if (entry.entryType === 'layoutJank') {
         this.aggregateJankScore_ += entry.fraction;
```
Comment (philipwalton) [inline src/service/performance-impl.js]: Be aware that this will calculate layout shifts even for cases that involved user input. Is that what you want?

**Comment by mrjoro** [comment]
Is this PR still active?

File: src/service/performance-impl.js (hunk 2 above)
Comment (ericandrewlewis) [reply src/service/performance-impl.js]: I think this is okay for this case. Chrome 75 versions are turning down, and we'll soon have the user interaction-aware metrics in Chrome 77 in about a month.

File: src/service/performance-impl.js (hunk 1 above)
Comment (ericandrewlewis) [reply src/service/performance-impl.js]: Thank you for catching this - added support for both versions in d569ea7f94acbfd062997def8d880b6351fb5727

**Comment by ericandrewlewis** [comment]
> Is this PR still active?

Yes, sorry for the delay in updating!

**Pull request merged**